docs/extensibility/extending-blocks.md

```diff
@@ -333,7 +333,7 @@ function my_plugin_block_categories( $categories, $post ) {
 			array(
 				'slug' => 'my-category',
 				'title' => __( 'My category', 'my-plugin' ),
-				'icon'  => 'screenoptions',
+				'icon'  => '<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="none" d="M0 0h24v24H0V0z" /><path d="M19 13H5v-2h14v2z" /></svg>',
```

Comment:
Small thing, but we also want to encourage block devs to include `width` and `height attributes.

Reply:
Icon component is used to render proper icons and current it enforces default width for dashicon and svg icons. Let me know if should add the width/height to example.

Reply:
If there's a default width that works I think this is fine.